dump/mapping/bufr_atms_mapping.yaml

```diff
@@ -0,0 +1,197 @@
+bufr:
+  variables:
+    # MetaData
+    timestamp:
+      datetime:
+        year: "*/YEAR"
+        month: "*/MNTH"
+        day: "*/DAYS"
+        hour: "*/HOUR"
+        minute: "*/MINU"
+        second: "*/SECO"
+
+    latitude:
+      query: "*/CLATH"
+
+    longitude:
+      query: "*/CLONH"
+
+    satelliteId:
+      query: "*/SAID"
+
+    satelliteInstrument:
+      query: "*/SIID"
+
+    fieldOfViewNumber:
+      query: "*/FOVN"
+
+    heightOfStation:
+      query: "*/HMSL"
+
+    solarZenithAngle:
+      query: "*/SOZA"
+
+    solarAzimuthAngle:
+      query: "*/SOLAZI"
+
+    sensorZenithAngle:
+      query: "*/SAZA"
+
+    sensorAzimuthAngle:
+      query: "*/BEARAZ"
+
+    sensorScanAngle:
+      sensorScanAngle:
+        fieldOfViewNumber: "*/FOVN"
+        scanStart: -52.725
+        scanStep: 1.110
+        sensor: atms
+
+    sensorChannelNumber:
+      query: "*/ATMSCH/CHNM"
+
+    # ObsValue
+    # Remapped Brightness Temperature
+    remappedBT:
+      remappedBrightnessTemperature:
+        fieldOfViewNumber: "*/FOVN"
+        sensorChannelNumber: "*/ATMSCH/CHNM"
+        brightnessTemperature: "*/ATMSCH/TMBR"
+        obsTime:
+          year: "*/YEAR"
+          month: "*/MNTH"
+          day: "*/DAYS"
+          hour: "*/HOUR"
+          minute: "*/MINU"
+          second: "*/SECO"
+
+  splits:
+    satId:
+      category:
+        variable: satelliteId
+        map:
+          _224: npp
+          _225: n20
+          _226: n21
+
+encoder:
+# type: netcdf
+
+  dimensions:
+    - name: Channel
+      source: variables/sensorChannelNumber
+      path: "*/ATMSCH"
+
+  globals:
+    - name: "platformCommonName"
+      type: string
+      value: "ATMS"
+
+    - name: "platformLongDescription"
+      type: string
+      value: "MTYP 021-203 ATMS ATENNA/BRIGHTNESS TEMPERATURE DATA"
+
+  globals:
+    - name: "platformCommonName"
```

Comment:
there are two globals called platformCommonName here. Also "globals" is here twice too.

Reply:
and two platformLongDescription.

Reply:
Extra definitions removed.